### PR TITLE
Make table level assertions have the same tags as the table

### PR DESCRIPTION
**Pull request by Ekrekr**
(with some automatic formatting fixes...)

### CODE REVIEW

File: core/table.ts [hunk 1]
```diff
@@ -664,7 +671,8 @@ export class Table {
       uniqueKeys.forEach((uniqueKey, index) => {
         const uniqueKeyAssertion = this.session.assert(
           `${this.proto.target.schema}_${this.proto.target.name}_assertions_uniqueKey_${index}`,
-          ctx => this.session.adapter().indexAssertion(ctx.ref(this.proto.target), uniqueKey)
+          ctx => this.session.adapter().indexAssertion(ctx.ref(this.proto.target), uniqueKey),
+          this.proto.tags
```
Comment (BenBirt) [inline core/table.ts:675]: this only works for when the caller calls `assertions(...)` *after* calling `tags(...)`.

consider the case where the caller adds another tag to the table *after* adding assertions. if you look at the equivalent for `disabled`, you will see that `disabled(...)` copies that through to any attached assertions - you need to do the same for tags.

**Review by BenBirt — CHANGES_REQUESTED** [review]

File: core/table.ts (hunk 1 above)
Comment (Ekrekr) [reply core/table.ts]: Done. I would edit a test for this case, but I don't see any pre-existing that test this order of events

**Review by BenBirt — APPROVED** [review]
Please update `version.bzl` to `2.1.0` and I will push a new version.

**Comment by Ekrekr** [comment]
> Please update `version.bzl` to `2.1.0` and I will push a new version.

Done.

**Pull request merged**